File: tests/fixtures/php/t105_violation.php
<?php

class MathTest extends \PHPUnit\Framework\TestCase
{
    public function test_add(): void
    {
        $this->assertEquals(4, Math::add(2, 2));
    }

    public function test_subtract(): void
    {
        $this->assertEquals(0, Math::subtract(2, 2));
    }

    public function test_multiply(): void
    {
        $this->assertEquals(6, Math::multiply(2, 3));
    }

    public function test_divide(): void
    {
        $this->assertEquals(2, Math::divide(4, 2));
    }

    public function test_modulo(): void
    {
        $this->assertEquals(1, Math::modulo(5, 2));
    }
}

File: tests/fixtures/rust/t105_violation.rs
#[test]
fn test_add() {
    assert_eq!(add(2, 2), 4);
}

#[test]
fn test_subtract() {
    assert_eq!(subtract(2, 2), 0);
}

#[test]
fn test_multiply() {
    assert_eq!(multiply(2, 3), 6);
}

#[test]
fn test_divide() {
    assert_eq!(divide(4, 2), 2);
}

#[test]
fn test_modulo() {
    assert_eq!(modulo(5, 2), 1);
}
